source/load-testing-dojot-platform.rst

```diff
@@ -35,10 +35,9 @@ Enter in Locust directory:
 Running a simple test
 ---------------------
 
-In this section, will be shown how to configure and run Locust and generate certs script in order
-to execute the load test. This is a simple test where will be created 100 clients to send messages
-to dojot.
-
+In this section, it will be shown how to configure and run Locust, and also how to generate 
+certificates with the generate_certs script in order to execute the load test. This is a simple 
```

Comment:
Suggestion

> certificates with the `generate_certs` script in order to execute

Use double backquotes \``text\`` for code samples
Note: _reStructuredText_ markup needs _two backquotes_ while the _markdown_ only needs _one_.

Reply:
Suggestion Accepted, thanks